Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.cs

```diff
@@ -233,7 +233,7 @@ private void RecalculateLayout(double containerWidth)
             if (containerWidth > 0)
             {
                 var newWidth = CalculateItemWidth(containerWidth);
-                ItemWidth = Math.Floor(newWidth - 1);
+                ItemWidth = Math.Floor(newWidth);
```

Comment:
Another issue with scrollbars.  Before:
![image](https://user-images.githubusercontent.com/24302614/35932780-7a8b9a54-0bed-11e8-9c35-03f71b0c94be.png)

After, the collapsible scrollbar is less noticeable as its margin isn't clear:
![image](https://user-images.githubusercontent.com/24302614/35933129-889c786a-0bee-11e8-8034-6332de7874fc.png)

Thoughts?

Reply:
That's standard for lists in which the items span the entire width. Here's a standard GridView with similar settings
![image](https://user-images.githubusercontent.com/1594689/35940618-ff4e208a-0c0c-11e8-85d2-28465c7f6e76.png)

Reply:
Good to know, I hadn't been paying as much attention to the overlap until this set of changes. :)